interview-questions.md

```diff
@@ -8,44 +8,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: to create a sub class
 
-Researched answer:
+Researched answer: Super is a keyword used to call a method of the same name from the parnet class of the current class.
```

Comment:
This is not quite accurate. [check here](https://github.com/learn-academy-2023-echo/syllabus/blob/main/ruby/inheritance.md)